### DISCUSSION TITLE
【委托】 救命！我快要进监狱了

### DISCUSSION THREAD

**Discussion by uyfh352** [share121/inter-knot#1010] [其他]
有没有大佬帮帮忙啊，9月1号就要进监狱了。真的好绝望，不知道怎么办。大佬刷到帖子的时候就给一些意见，劫狱还是啥的都好，有没有办法不进监狱。
对了我是学生党，没有委托费
![Screenshot_2024-08-10-20-51-35-238_com android chrome-edit](https://github.com/user-attachments/assets/281accd5-23ff-4928-98d4-666597289c5d)

**Comment by tuhate2** [comment]
去当原神内鬼然后就能被抓去当50万，避免监狱的悲惨结局🤓

**Comment by Qiling-XwX** [comment]
这边建议找个大运，躲进异世界
![66E99D86FD12133F8466402D24D65280](https://github.com/user-attachments/assets/d42c378c-3739-4f75-8216-c5604ed219d4)

**Comment by shuyishaoxiancao** [comment]
这委托爱谁接谁接

**Comment by hanshui2077** [comment]
同为狱友，我要去找大运去异世界找雷姆了
![c0b418d39a3444e997791a823bc6bea3](https://github.com/user-attachments/assets/0a0efd91-7e18-4430-8409-8dbc34677944)
